src/ChannelServer/Util/GmCommands.Handlers.cs

```diff
@@ -662,6 +663,60 @@ private CommandResult HandleRecall(ChannelConnection conn, Character sender, Cha
 		}
 
 		/// <summary>
+		/// Warps all players on the map to target's location.
+		/// </summary>
+		/// <param name="conn"></param>
+		/// <param name="sender"></param>
+		/// <param name="target"></param>
+		/// <param name="command"></param>
+		/// <param name="args"></param>
+		/// <returns></returns>
+		private CommandResult HandleRecallMap(ChannelConnection conn, Character sender, Character target, string command, string[] args)
+		{
+			if (args.Length > 2)
+				return CommandResult.InvalidArgument;
+
+			Character[] characters = null;
+			
+			// Recall same map.
+			if (args.Length == 1)
```

Comment:
Looks good, but I was wondering what you think about how I would've done it.
```csharp
var map = sender.Map;
if (args.Length > 1)
{
	// Search for map by name and id
	if (int.TryParse(args[1], out var mapId))
		map = ChannelServer.Instance.World.GetMap(mapId);
	else
		map = ChannelServer.Instance.World.GetMap(args[1]);

	// Check map
	if(map == null)
	{
		sender.ServerMessage("Unknown map.");
		return CommandResult.Okay;
	}
}

var characters = map.GetCharacters(a => a != target);
```

Reply:
Pushed, looks good for me.

Just don't forget the comment to check for max. player count. ^^